src/camera/linux/LinuxCamera.zig

```diff
@@ -0,0 +1,13 @@
+const std = @import("std");
+
+pub const LinuxCamera = struct {
+    pub fn getFrame(self: LinuxCamera) [*:0]const u8 {
+        _ = self;
+        return @embedFile("../test.raw");
```

Comment:
Only using an `@embedFile` for now but this is where the actual frame should be read from the camera on Linux.